package.json

```diff
@@ -47,15 +44,16 @@
         "react-dom": "^17.0.2",
         "react-helmet": "^6.1.0",
         "react-i18next": "^11.15.3",
-        "react-is": "^17.0.2",
```

Comment:
dont we need this one anymore?

Reply:
I searched the whole project, it's not used anywhere. should we keep it?

Reply:
no, its ok,
lets remove it